resources/models.py

```diff
@@ -314,11 +316,26 @@ def get_queryset(self):
             berth_qs, place_type_name="berth_type", section_name="pier"
         )
 
+        # When annotating the count, if no elements match the filter, the whole queryset will
+        # return an empty QuerySet<[]>, which will then return None as value for the count.
+        #
+        # By adding the other annotate with Coalesce, we ensure that, we'll always get an int value
         available_berths = (
             berth_qs.filter(is_available=True, is_active=True)
             .order_by()
             .values("pier")
-            .annotate(count=Count("*"))
+            .annotate(nullable_count=Count("*"))
+            .values("nullable_count")
+            .annotate(count=Coalesce("nullable_count", Value(0)))
+            .values("count")
+        )
+        unavailable_berths = (
```

Comment:
I would use `inactive` for this, since our `is_available` marks whether there is a lease or not, but here we mark those that are not active

Reply:
So it would be the same for the annotation/field? `number_of_inactive_places` instead of `number_of_unavailable_berths`?

Reply:
yeah, I think `number_of_inactive_places` is more explicit about its value.

Reply:
Right, makes more sense since we don't actually check for `unavailable` but `inactive` places. 👍

Reply:
![image](https://user-images.githubusercontent.com/24206160/87930669-27802680-ca91-11ea-9405-9853f1a4d72f.png)